### PR TITLE
Improve style of GWLF-E BMP UI

### PR DESCRIPTION
**Pull request by lewfish**
This improves the style of the GWLF-E BMP UI. To test, make sure functionality is intact and things are more aesthetically pleasing. Compare to https://github.com/WikiWatershed/model-my-watershed/pull/1314

* Make style of Animal Waste Management (Poultry) slightly different from (Livestock)
* Add Rural and Urban thumbnail groups
* Add more vertical space in thumbnail view so labels are fully visible
* For BMPs with multiple data model variables, change the label of the input to clarify which data model variable is being converted
* In manual entry view: change size and color of info and error messages, improve spacing, and put fill pattern in line under header to improve readability

screen shot 2016-05-23 at 3 31 29 pm
screen shot 2016-05-24 at 5 29 57 pm

Connects #1315

### CODE REVIEW

**Comment by mmcfarland** [comment]
I think the visual style of the display bullet points would be better as a table (perhaps with no border) and condense the verbiage.

| Length of streams in Ag areas | 300km |
| ---      | ---       |
| **Length of steams in Watershed** | **1000km** |

Also, "Convert" is a holdover from when we though were were merely converting a land use type into a BMP, although now understanding how it actually works I think "Apply" is better suited.

**Comment by lewfish** [comment]
> I think the visual style of the display bullet points would be better as a table (perhaps with no border) and condense the verbiage.

Done. See updated screenshot.

**Comment by lewfish** [comment]
> Also, "Convert" is a holdover from when we though were were merely converting a land use type into a BMP, although now understanding how it actually works I think "Apply" is better suited.

I think 'Modify' might be a better replacement for 'Convert' because it makes more sense in string names and internal variables. For instance, 'Length of stream to modify' rather than 'Length of stream to apply', and `lengthToModify` rather than `lengthToApply.` We could use 'modify' in those places, but still use 'apply' on the button itself.

**Comment by caseypt** [comment]
+1. Looks more aesthetically pleasing.

File: src/mmw/js/src/modeling/controls.js [hunk 1]
```diff
@@ -111,8 +111,22 @@ var InputInfoView = Marionette.ItemView.extend({
     },
 
     templateHelpers: function() {
+        var output = this.model.get('output'),
+            errorMessage = output && output.errorMessages[this.userInputName],
+            infoMessage = output && output.infoMessages[this.userInputName],
```
Comment (caseypt) [inline src/mmw/js/src/modeling/controls.js]: I did not know that using `&&` in this way would return the correct value here. Pretty interesting.

**Pull request merged**